docs/src/test-api/class-testconfig.md

```diff
@@ -629,14 +629,15 @@ export default config;
   - `reuseExistingServer` ?<[boolean]> If true, it will re-use an existing server on the `port` or `url` when available. If no server is running on that `port` or `url`, it will run the command to start a new server. If `false`, it will throw if an existing process is listening on the `port` or `url`. This should be commonly set to `!process.env.CI` to allow the local dev server when running tests locally.
   - `cwd` ?<[string]> Current working directory of the spawned process, defaults to the directory of the configuration file.
   - `env` ?<[Object]<[string], [string]>> Environment variables to set for the command, `process.env` by default.
+  - `setBaseURL` ?<[boolean]> If false, [`property: TestOptions.baseURL`] will not be automatically set based on `port`. See docs for more details on how [`property: TestOptions.baseURL`] precedence works when using `webServer`.
```

Comment:
In the new world, I'd like it to be opt-in. I also don't think we are solving a use case when baseURL depends on the port but does not match it. So I wonder if we could avoid adding new stuff into webServer config for now.